src/picomsg/src/oper_db_client.h

```diff
@@ -19,6 +19,12 @@ struct oper_db_client_db_obj_t {
 
 	bool (*enum_set)(oper_db_client_db_obj_t* _this, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, enum LIBOPER_DB_CLIENT_FIELD_ENUM_VALUES enum_value);
 	bool (*u64_set)(oper_db_client_db_obj_t* _this, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint64_t new_value);
+	bool (*u16_set)(oper_db_client_db_obj_t* _this, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint16_t new_value);
+	bool (*u8_set)(oper_db_client_db_obj_t* _this, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint8_t new_value);
+	bool (*bool_set)(oper_db_client_db_obj_t* _this, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint8_t new_val);
```

Comment:
`new_val` needs to be a bool. Please check the other `bool_set` functions and fix them accordingly if they also use `uint8_t`.